Исправитель текстовых и номерных ошибок. Этап - Тим Лидер

Ivanstarterprog opened the pull request.

Ivanstarterprog committed:
Написана логика класса-исправителя текстовых и номерных ошибок

Ivanstarterprog committed:
Добавлен класс меню с паттерном Синглтон

Hamjoshua commented on Program.cs:
```diff
@@ -0,0 +1,166 @@
+
+    class Menu
+    {
+        private static Menu instance;
```
Не хватает _

Hamjoshua commented on Program.cs:
```diff
@@ -0,0 +1,166 @@
+                    MenuChangeDictionary();
+                    break;
+                default:
+                    Console.WriteLine("Неопознанная команда! Попробуйте заного");
```
*заново

Hamjoshua commented on Program.cs:
```diff
@@ -0,0 +1,166 @@
+            using (StreamReader streamReader = new StreamReader(textFile))
+            {
+                string line;
+                while((line = streamReader.ReadLine()) != null)
```
не хватает пробела: while ((...))

Ivanstarterprog committed:
Исправление ошибок
1. Исправлены названия переменных
2. Исправлен Code Style
3. Исправлена ошибка правописания (Разработчик получил по голове словарём Русского языка)
